safe/utilities/keyword_io.py

```diff
@@ -670,3 +678,83 @@ def get_statistics(self, layer):
             statistics_classes = {}
 
         return statistics_type, statistics_classes
+
+
+def write_to_new_keyword(source, keywords):
```

Comment:
Yes, I will. Thanks. I just named it like this to experiment.
There are so many to do, including updating the old data tests.